Automattic/sensei
# Make course list inner blocks available on all screens

Implements https://github.com/Automattic/sensei/issues/5447

### Changes proposed in this Pull Request

* Made the Course List block and the inner blocks available everywhere for all post types. The blocks are -
   - View Results
   - Course Signup
   - Course Progress
   - Course Categories
   - Course Actions

### Testing instructions
- Go to every GB editor pages, like course editor, page editor, post editor etc
- Add the course list block, then try adding the other blocks inside it (Inner blocks are not expected to work unless they are on the course editor page or inside Course List block's inner template)
- Make sure you can add all the listed blocks
- Make sure all of them work and appear with all the styles in the frontend
- Make sure the styles don't break both in the back and front end
- Go to other normal pages without sensei blocks in the front end and make sure the styles don't break
- Try different sensei and non-sensei blocks in the GB editor and make sure non of them breaks
- Try these with different themes

### Screenshot / Video

https://user-images.githubusercontent.com/6820724/185008737-d8d2e44b-f60f-4ad2-927d-f45e1fa93251.mov
